tests/cli/test_diff.py

```diff
@@ -16,8 +16,16 @@
 from datumaro.util.image import Image
 from datumaro.util.test_utils import TestDir
 
+import pytest
+from tests.requirements import Requirements
+from tests.requirements import DatumaroComponent
```

Comment:
```suggestion
from tests.requirements import Requirements, DatumaroComponent
```

Reply:
Maxim, where do you prefer to put requirements.py file, in main directory or in tests directory?

Reply:
In the test directory

Reply:
Corrected